src/plone/passwordless/profiles/default/registry.xml

```diff
@@ -0,0 +1,4 @@
+<records prefix="plone.resources/plone.passwordless"
+         interface='Products.CMFPlone.interfaces.IResourceRegistry'>
+    <value key="js">++plone++plone.passwordless/script.js</value>
```

Comment:
We need to create a bundle at some point later in the project. For now it is not necessary since we call the `script.js` directly from within the template. Leave your `registry.xml` file for now but please remove the line with the `value`.